### ISSUE TITLE
Feature: Retrieve historical data

### ISSUE THREAD

**Issue by bsmulders** [mendhak/gpslogger#1069]
I have used gpslogger for a couple of days during a trip. It would be nice if I could retrieve the GPS logs for those days and import them into OsmAnd to view and analyse the data.

**Comment by mendhak** [comment]
I'm not super familiar with OsmAnd, only lightly used it. If you've recorded GPX files then you should be able to import those files under My Places > Tracks tab.  I also tried from X-plore file explorer, long pressed the GPX and opened with OsmAnd.

**Comment by vincele** [comment]
Yes, I can confirm osmand being able to open and show an exported GPX file coming from gpslogger. I use that feature all the time since years for my hike & bike recordings. And as @mendhak said, you can open the GPX track from within osmand or from a file explorer (I use the standard `File` app from LineageOS).

Maybe this issue can be closed ?
